src/content/docs/query-your-data/nrql-new-relic-query-language/get-started/nrql-syntax-clauses-functions.mdx

```diff
@@ -503,7 +503,11 @@ Every NRQL query will begin with a `SELECT` statement or a `FROM` clause. All ot
 
     The **default** value is **1 hour ago**.
 
-    Use the `SINCE` clause to define the beginning of a time range for the returned data. When using NRQL, you can set a UTC timestamp or relative time range. You can specify a timezone for the query but not for the results. NRQL results are based on your system time.
+    Use the `SINCE` clause to define the beginning of a time range for the returned data. You can specify a timezone for the query but not for the results. NRQL results are based on your system time.
+
+    When using NRQL, you can set a UTC timestamp or a relative time range:
+    - Timestamps use the format `YYYY-MM-DD HH:MM:SS ZZZZ`. For instance, `FROM Transaction SELECT count(*) SINCE '2021-12-25 00:00:00 +0000' UNTIL '2021-12-25 23:59:59 +0000'`.
+    - We support the following relative time ranges: `NOW`, `YESTERDAY`, `TODAY`, `SUNDAY`, `MONDAY`, `TUESDAY`, `WEDNESDAY`, `THURSDAY`, `FRIDAY`, `SATURDAY`, `LAST`, `THIS YEAR`, `QUARTER`, `MONTH`, `WEEK`, `DAY`, `HOUR`, `MINUTE`, `SECOND`.
```

Comment:
Hi @urbiz-nr , thanks for writing this up.

This isn't quite correct. LAST and THIS are only valid on a subset of the relative time ranges. For example, with the following (`YESTERDAY | TODAY |  SUNDAY |  MONDAY |  TUESDAY |  WEDNESDAY |  THURSDAY |  FRIDAY |  SATURDAY`), you'd write something like `SINCE  YESTERDAY UNTIL NOW`

For these relative time ranges (`YEAR | QUARTER | MONTH | WEEK | DAY | HOUR | MINUTE | SECOND`), you'll need to include either `THIS` or `LAST`, something like `SINCE LAST WEEK UNTIL NOW` or event `SINCE LAST MONTH UNTIL THIS WEEK`, etc.

Reply:
Thanks @mgmur - adding this!